### DISCUSSION TITLE
[@types/express-slow-down] `SlowDown` `headers` missing.

### DISCUSSION THREAD

**Discussion by xCykrix** [DefinitelyTyped/DefinitelyTyped#55991] [Issues with a @types package] [answered]
`headers` option missing from `SlowDown.options`
Reference: https://github.com/nfriedly/express-slow-down/blob/master/lib/express-slow-down.js#L69

> headers: Add X-SlowDown-Limit, X-SlowDown-Remaining, and if the store supports it, X-SlowDown-Reset headers to all responses. Modeled after the equivalent headers in express-rate-limit. Default: false

[a comment by an automated account is not shown]

**Comment by jdforsythe** [comment, marked as answer]
Fixed by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56010

**Reply by xCykrix** [reply]
Thank you! I just didn't want to have to use my jank workaround forever of ts-expect-error haha.
